src/Dofs/MixedDofHandler.jl

```diff
@@ -1,15 +1,30 @@
+"""
+Field(name::Symbol, interpolation::Interpolation, dim::Int)
```

Comment:
```suggestion
    Field(name::Symbol, interpolation::Interpolation, dim::Int)
```
Noticed formatting missing while reading the preview doc...